src/mame/sega/segaufo.cpp

```diff
@@ -533,56 +533,96 @@ static INPUT_PORTS_START( newufo )
+	PORT_DIPSETTING(	0x90, DEF_STR( 1C_8C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x80, "1 Coin/10 Credits" )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x70, "1 Coin/11 Credits" )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x60, "1 Coin/12 Credits" )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x50, DEF_STR( 2C_1C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x40, DEF_STR( 2C_3C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x30, DEF_STR( 3C_1C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x20, DEF_STR( 3C_2C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x10, DEF_STR( 4C_1C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
+	PORT_DIPSETTING(	0x00, DEF_STR( 5C_1C ) )	PORT_CONDITION("DSW2",0x08,EQUALS,0x00)
 
 	PORT_START("DSW2")
-	PORT_DIPNAME( 0x01, 0x01, "UNK2-01 Demo Music Off" )
-	PORT_DIPSETTING(    0x01, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x02, 0x02, "UNK2-02" )
-	PORT_DIPSETTING(    0x02, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x04, 0x04, "UNK2-04 Initial Motor Test" )
+	PORT_DIPNAME( 0x01, 0x01, DEF_STR( Demo_Sounds ) )				PORT_DIPLOCATION("SW2:1") // Manual states "Sounds"
+	PORT_DIPSETTING(    0x01, DEF_STR( On ) )						                          // Manual states "Always"
+	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )                                                // Manual states "Only During Play"
+	PORT_DIPNAME( 0x02, 0x02, "Arm X, Y-Move" )						PORT_DIPLOCATION("SW2:2")
+	PORT_DIPSETTING(    0x02, "Each Once" )
+	PORT_DIPSETTING(    0x00, "Limitless" )
+	PORT_DIPNAME( 0x04, 0x04, "Initial Mechanism Motion Check" )	PORT_DIPLOCATION("SW2:3")
 	PORT_DIPSETTING(    0x04, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x08, 0x08, "UNK2-08" )
-	PORT_DIPSETTING(    0x08, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x10, 0x10, "UNK2-10 Disable Prize Sensor" )
+	PORT_DIPNAME( 0x08, 0x08, DEF_STR( Coinage ) )					PORT_DIPLOCATION("SW2:4")
+	PORT_DIPSETTING(    0x08, "SETTING 1" )
+	PORT_DIPSETTING(    0x00, "SETTING 2" )
+	// Manual states "5 ~ 8 must be set OFF."
+	PORT_DIPNAME( 0x10, 0x10, "Avoid Prize Sensor Error" )			PORT_DIPLOCATION("SW2:5")
 	PORT_DIPSETTING(    0x10, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x20, 0x20, "UNK2-20" )
-	PORT_DIPSETTING(    0x20, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x40, 0x40, "UNK2-40" )
-	PORT_DIPSETTING(    0x40, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x80, 0x80, "UNK2-80" )
-	PORT_DIPSETTING(    0x80, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
+	PORT_DIPSETTING(    0x00, DEF_STR( On ) ) // Manual states "Fanfare and CPanel Digit will not work though."
```

Comment:
Sorry for erasing it so many times.
The original wording is "When DIP SW 2 #5 is set to ON, the game can be played normally without error even if the prize sensor is malfunctioning.(DIP SW 2 #5 を ON にすると、プライズセンサーが動作不良の場合でもエラーにならずに通常通りゲームを行えます。)"
It might as well write "Ignore Prize Sensor Error" or "Ignore Prize Sensor malfunction".

Reply:
It’s better to just call it “Ignore Prize Sensor”.  That explains what it does.  I’ll patch up this PR and merge it.